Sources/SentrySwiftUI/SentryReplayView.swift

```diff
@@ -3,26 +3,41 @@ import Sentry
 import SwiftUI
 import UIKit
 
+#if CARTHAGE || SWIFT_PACKAGE
+@_implementationOnly import SentryInternal
+#endif
+
+enum MaskBehaviour {
```

Comment:
I don't know about the locale on your machine, but for me this displays as a spellcheck error. I will say that American english spells it without the `u`, and we already have `behavior` all over the place in the codebase; introducing an alternative spelling hurts greppability.

```suggestion
enum MaskBehavior {
```

Reply:
I dont know why my machine uses UK English.